src/plugins/multipart.js

```diff
@@ -159,8 +159,9 @@ function createInitMultipart(boundary) {
               ),
             );
         }
-
-        this.onPart(part);
+        this._parser.pause();
```

Comment:
This **might** hurt performance a lot (I am really not sure, testing needs to be done). If performance would be to be (at least partially, because the async function is always a bit slower) regained, I propose the following options:
A. Instead of waiting for the folder and file to be written, wait only if the previous file was not yet processed; it will very probably be written by the time, the next file comes around.
B. Don't change a thing on how files are written down and instead move them after the upload, or even better, only tell others that the file is supposed to be in a folder.

Reply:
Yes I was thinking about it. I will do some performance tests

Reply:
I tested this live for more than 5 months and users did not notice any slow downs,
Today I ran an e2e mass request test and the result show that performance impact is lower than the margin of error:

bombardier --body-file="./README.md" --method=POST 
--duration=10s --connections=100 http://localhost:3000/api/upload 
Bombarding http://localhost:3000/api/upload for 10s using 100 connection(s)[====================================================================] 10s 
Done!
Statistics        Avg      Stdev        Max
  Reqs/sec      3173.09    1221.25    6372.28
  Latency       31.55ms    12.85ms   320.89ms
  HTTP codes:
    1xx - 0, 2xx - 31765, 3xx - 0, 4xx - 0, 5xx - 0
    others - 0
  Throughput:   116.28MB/s
  
  See https://github.com/node-formidable/formidable/blob/master/benchmark/2022-11-30-i5-9600k.txt for the result I got on master. So a bit faster in this case. If I rerun this multiple times I sometimes get results in the 88-120.MB/s range